src/Streamly/Internal/Prelude.hs

```diff
@@ -3506,4 +3506,4 @@ bracket bef aft bet = D.fromStreamD $
 handle :: (IsStream t, MonadCatch m, Exception e)
     => (e -> t m a) -> t m a -> t m a
 handle handler xs =
-    D.fromStreamD $ D.handle (\e -> D.toStreamD $ handler e) $ D.toStreamD xs
+    D.fromStreamD $ D.handle (\e -> D.toStreamD $ handler e) $ (D.toStreamD xs)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/composewell/streamly/blob/6dfbd1dfb23967476bdfef206c007b08e8eac20c/src/Streamly/Internal/Prelude.hs#L3509"></span>Found Avoid lambda</p>

<pre><code class="language-haskell">\ e -&gt; D.toStreamD $ handler e
</code></pre>

<p>Why Not</p>

<pre><code class="language-haskell">D.toStreamD . handler
</code></</td>
    </tr>
  </tbody>
</table>